rotkehlchen/api/v1/schemas.py

```diff
@@ -1606,11 +1606,15 @@ class ModuleHistoryProcessingSchema(HistoryProcessingSchema):
     module = SerializableEnumField(enum_class=ModuleWithStats, required=True)
 
 
-class HistoryProcessingExportSchema(HistoryProcessingSchema):
+class AsyncDirectoryPathSchema(AsyncQueryArgumentSchema):
     directory_path = DirectoryField(load_default=None)
 
 
-class HistoryProcessingDebugImportSchema(AsyncQueryArgumentSchema):
+class HistoryProcessingExportSchema(AsyncDirectoryPathSchema, TimestampRangeSchema):
+    """Schema for exporting history events"""
+
+
+class AsyncFilePathSchema(AsyncQueryArgumentSchema):
     filepath = FileField(required=True, allowed_extensions=['.json'])
```

Comment:
can you create this one after `AsyncDirectoryPathSchema` so things that are related are near